content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst

```diff
@@ -0,0 +1,108 @@
+.. _plm/eco/eco-type:
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+.. |ECO| replace:: :abbr:`ECO (Engineering Change Order)`
+.. |ECOs| replace:: :abbr:`ECOs (Engineering Change Orders)`
+
+========
+ECO type
+========
+
+Organize engineering change orders (ECOs) into a *ECO type* project and create custom stages to
```

Comment:
We typically use *italics* for initialisms, not capital letters.

The italics are more appropriate, I'd argue, because:
- stylistically, they are is as effective as capital letters in visually communicating a difference in the text, meaning, they clue in the reader to pay attention;
- orthographically, they are correct since they are actually emphasizing something, like italics is supposed to do. Whereas, the capitalization is a bit more of a subjective use case in stylizing the initialisms (they're not proper nouns, after all), no?.
- consistently, this is what we typically do on doc for acronym,  so whether or not it's the best or most correct way, it is at least...consistent.

Whether or not we agree on this, I appreciate you for giving me the excuse to flex _orthography_— that one had some dust on it! 😅 These, by the way, are definitely the nuanced ideas/discussions I like to see, so cheers to that! 🍻